packages/syft/src/syft/serde/recursive.py

```diff
@@ -115,7 +117,7 @@ def combine_bytes(capnp_list: List[bytes]) -> bytes:
     return bytes_value
 
 
-def rs_object2proto(self: Any) -> _DynamicStructBuilder:
+def rs_object2proto(self: Any, for_hashing: bool = False) -> _DynamicStructBuilder:
```

Comment:
This is the part where preference of attrs for hashing for each nested objects will be taken into consideration.